src/test/fixtures/profiles/processed-profile.js

```diff
@@ -1696,14 +1696,14 @@ export function addActiveTabInformationToProfile(
     },
     // An iframe page inside the previous page
     {
-      tabID: 2,
+      tabID: firstTabTabID,
```

Comment:
I think this was a mistake at first, right? Because the `innerWindowID` specified below are part of the first tab.

Reply:
Ah, yeah. I don't know how this ended up like this, but they should be the same indeed. 

Hm, maybe it's because previously we weren't using tab ids, we were using browsingContextIds instead. They were all different. It might be that these are leftover from that work. Thanks for the fix.